packages/core/src/browser/menu/browser-menu-plugin.ts

```diff
@@ -156,6 +160,10 @@ export class BrowserMainMenuFactory implements MenuWidgetFactory {
 
 }
 
+export function isMenuElement(element: HTMLElement | null): boolean {
+    return !!element && element.className.includes('lm-Menu');
+}
```

Comment:
Question: Is this required? It does not seem to be in use and also uses Lumino stuff.

Reply:
Good catch, I ported this commit from the lumino-PR.